tests/v2/test_problem.py

```diff
@@ -18,8 +16,11 @@ def test_load_remote():
 
     assert petab_problem.validate() == []
 
-    yaml_url = yaml_url.replace("2.0.0", "1.0.0")
-    with pytest.raises(
-        ValueError, match="Provided PEtab files are of unsupported version"
-    ):
-        Problem.from_yaml(yaml_url)
+
+def test_auto_upgrade():
+    yaml_url = (
+        "https://raw.githubusercontent.com/PEtab-dev/petab_test_suite"
+        "/main/petabtests/cases/v1.0.0/sbml/0001/_0001.yaml"
+    )
+    problem = Problem.from_yaml(yaml_url)
+    assert isinstance(problem, Problem)
```

Comment:
:sweat_smile:
```suggestion
    problem = Problem.from_yaml(yaml_url)
    # TODO check something specifically different in a v2 problem
    assert isinstance(problem, Problem)
```

Reply:
Yes... can check more once v2 specs are closer to final.